examples/java/pom.xml

```diff
@@ -48,6 +48,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <trimStackTrace>false</trimStackTrace>
```

Comment:
okay, I think trimStackTrace is already set to false, inherited from parent POM. 
Verified with "mvn help:effective-pom" in HEAD.

But, Jenkins doesn't show the exception cause/chain.
https://builds.apache.org/job/beam_PreCommit_MavenVerify/org.apache.beam$beam-examples-java/1584/testReport/junit/org.apache.beam.examples/WordCountIT/testE2EWordCount/

And, the log is truncated for passed tests.

...[truncated 10915 bytes]...

https://builds.apache.org/job/beam_PreCommit_MavenVerify/org.apache.beam$beam-examples-java/1600/testReport/org.apache.beam.examples/WordCountIT/testE2EWordCount/

I searched a little bit, didn't find an answer. I am closing this PR.